File: Backend/AppCode/AppConstants.cs
namespace Backend.AppCode
{
    public static class AppConstants
    {
        public static class UserRole
        {
            public const string Admin = "Admin";
            public const string User = "User";
        }
        public enum InvestmentType
        {
            Stocks, Bonds, RealEstate, Crypto, MutualFunds, Other
        }
        public enum InvestmentStatus
        {
            Active, Sold, OnHold
        }
        public enum TransactionType
        {
            Buy, Sell, Update
        }
        public enum EntityType
        {
            User, Portfolio, Investment, Transaction
        }
        public enum ActivityAction
        {
            Create, Update, Delete, Login, Logout, Export, Seed
        }

    }
}

File: Frontend/src/constants/appConstants.ts
const APP_NAME = 'Investment Portfolio Management System';
const APP_SHORT_NAME = 'IPMS';

// Investment Types (matching backend enum)
export const INVESTMENT_TYPES = [
  'Stocks',
  'Bonds',
  'RealEstate',
  'Crypto',
  'MutualFunds',
  'Other',
] as const;

// Investment Statuses (matching backend enum)
export const INVESTMENT_STATUSES = ['Active', 'Sold', 'OnHold'] as const;

// Transaction Types (matching backend enum)
export const TRANSACTION_TYPES = ['Buy', 'Sell', 'Update'] as const;
